### DISCUSSION TITLE
Show Hover Card based on hovered element for dynamic data display

### DISCUSSION THREAD

**Discussion by maulik13** [chakra-ui/chakra-ui#9391] [Q&A] [answered]
I have a calendar component that shows dates in a timeline with each date holding some data for that day. I want to show a hover card with data for that date. One option is to render each individual date in individual <HoverCardTrigger> per Hover Card. But this seems like unnecessary duplication of a components while only one will be displayed and only the data is different. 

I have managed to create a controlled single Hover Card that picks up data for the current hovered date and displays it. However, the position is wrong obviously. How would I go about using Hover Card in this case?

**Comment by segunadebayo** [comment]
Do you have a reproduction?

**Comment by segunadebayo** [comment]
Use one `HoverCard.Root`. Point it at the hovered cell with `positioning.getAnchorRect`.

Do not mount a HoverCard per date if only the payload changes.

**Comment by segunadebayo** [comment, marked as answer]
Same pattern on Hover Card. Put `value` on each `HoverCard.Trigger`.

https://chakra-ui.com/docs/components/hover-card#multiple-triggers

`getAnchorRect` is still the right tool for a cell that is not a trigger.
